elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -252,7 +252,7 @@ class PfeContentSet extends PFElement {
     this._updateBreakpoint = this._updateBreakpoint.bind(this);
 
     this._observer = new MutationObserver(this._mutationHandler);
-    if (window.ResizeObserver) this._resizeObserver = new ResizeObserver(this._resizeHandler);
+    // if (window.ResizeObserver) this._resizeObserver = new ResizeObserver(this._resizeHandler);
```

Comment:
This bad-boy was failing

Reply:
Can you add comments around the commented out code sections referencing the Webkit bug?